app/modules/ads/setup.js

```diff
@@ -196,10 +196,17 @@ export const adsSetup = {
     context.set('slots.floor_adhesion.disabled', !instantConfig.isGeoEnabled('wgAdDriverMobileFloorAdhesionCountries'));
     if (instantConfig.get('icFloorAdhesionClickPositionTracking')) {
       context.set('slots.floor_adhesion.clickPositionTracking', true);
```

Comment:
We can move this line above L205 and get rid of this if statement.

Reply:
💎